setup.py

```diff
@@ -37,9 +38,14 @@ class build_py(_build_py):
     def pin_version(self):
         path = os.path.join(self.build_lib, "dvc")
         self.mkpath(path)
-        with open(os.path.join(path, "version.py"), "w") as fobj:
-            fobj.write("# AUTOGENERATED at build time by setup.py\n")
-            fobj.write(f'__version__ = "{version}"\n')
+        (Path(path) / "version.py").write_text(
+            dedent(
+                """\
+                # AUTOGENERATED at build time by setup.py
+                fobj.write(f'__version__ = "{version}"
```

Comment:
Missing closing parentheses
```suggestion
                fobj.write(f'__version__ = "{version}"')
```

Reply:
no just missing `Ctrl`+`S` :D